templates/zerver/integrations.html

```diff
@@ -547,6 +553,35 @@
       <img class="screenshot" src="/static/images/integrations/codebase/001.png" />
     </div>
 
+    <div id="codeship" class="integration-instructions">
+
+      <p>Zulip supports integration with Codeship and can notify you of
+          your build statuses.</p>
+
+      <p>First, create the stream you'd like to use for Codeship notifications,
+          and subscribe all interested parties to this stream. We
+          recommend the name <code>codeship</code>.</p>
+
+      <p><code>{{ external_api_uri }}/v1/external/codeship?api_key=abcdefgh&amp;stream=codeship</code></p>
+
+      <p>where <code>api_key</code> is the API key of your Zulip bot,
+      and <code>stream</code> is the stream name you want the
+      notifications sent to.</p>
+
+      <p>Next, go to your project settings - notification settings - url should look like the following </p>
+      <p><code>https://codeship.com/projects/PROJECT_ID/configure_notifications</code></p>
+      <p>where <code>PROJECT_ID</code>is id of your project in Codeship</p>
+      <p>scroll to the <code>Webhook</code> section and fill out form as follows:</p>
+      <img class="screenshot" src="/static/images/integrations/codeship/001.png" />
+
+      <p>Remember about enabling your integration</p>
```

Comment:
It regards enable switch on the picture. Maybe it's unnecessary.
What do you think?